src/data/versions/dev6.0.41r-1.json

```diff
@@ -32,10 +32,10 @@
             ]            
         },
         "ER Settings": {
-            "open": true,
```

Comment:
I would prefer to keep ER options expanded by default as the default settings on page load is full ER. Less clicks to tone it down if not using a preset.

Reply:
for sure, some tweaks i made mostly to my own personal preferences may have crept in, can be reverted easily enough